staging/src/k8s.io/client-go/tools/cache/controller_test.go

```diff
@@ -893,3 +906,177 @@ func (m *mockTxnStore) Transaction(txns ...Transaction) *TransactionError {
 	m.succeedCount = len(txns)
 	return nil
 }
+
+func TestAtomicReplaceEquivalence(t *testing.T) {
+	_, ctx := ktesting.NewTestContext(t)
+	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
+
+	type eventRecord struct {
+		Action string
+		Key    string
+	}
+
+	runControllerScenario := func(t *testing.T, enabled bool) ([]interface{}, []eventRecord) {
```

Comment:
Can you move this to a normal function? Standard convention is to have lowercase test name. In this case `testAtomicReplaceEquivalence`